Non-ASCII characters cause table column width to grow unnecessarily

See https://github.com/sherter/ghokin/commit/ea0fba40560883a7fa921f6593107da2aa99c80b

For each non-ascii character in the table cell, the table column width grows one space larger than it needs to be.

```
    --- FAIL: TestTransform/fixtures/table-with-non-ascii-characters.feature (0.00s)
        transformer_test.go:475:
                Error Trace:    /home/simon/repositories/ghokin/ghokin/transformer_test.go:475
                Error:          Not equal:
                                expected: "Feature: A Feature\n  Description\n\n  Scenario: A scenario to test\n    Given a thing\n      | äöüûú |\n\n"
                                actual  : "Feature: A Feature\n  Description\n\n  Scenario: A scenario to test\n    Given a thing\n      | äöüûú      |\n\n"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -5,3 +5,3 @@
                                     Given a thing
                                -      | äöüûú |
                                +      | äöüûú      |

                Test:           TestTransform/fixtures/table-with-non-ascii-characters.feature
```

In this example we have 5 non-ascii characters and the column is 5 spaces larger than it needs to be.

--- a/ghokin/fixtures/non-ascii-characters-formatting.feature
+++ b/ghokin/fixtures/non-ascii-characters-formatting.feature
@@ -1,0 +1,5 @@
+Feature: Test
+
+  Scenario: This is a scenario
+    Given I have a table:
+      | äöüûú |

--- a/ghokin/transformer.go
+++ b/ghokin/transformer.go
@@ -7,6 +7,7 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+	"unicode/utf8"
 
 	"github.com/cucumber/common/gherkin/go/v23"
 )
@@ -244,9 +245,9 @@ func calculateLonguestLineLengthPerColumn(rows [][]string) []int {
 		for j, str := range row {
 			switch true {
 			case i == 0:
-				lengths = append(lengths, len(str))
-			case i != 0 && len(lengths) > j && lengths[j] < len(str):
-				lengths[j] = len(str)
+				lengths = append(lengths, utf8.RuneCountInString(str))
+			case i != 0 && len(lengths) > j && lengths[j] < utf8.RuneCountInString(str):
+				lengths[j] = utf8.RuneCountInString(str)
 			default:
 				lengths = append(lengths, 0)
 			}

--- a/ghokin/transformer_test.go
+++ b/ghokin/transformer_test.go
@@ -444,6 +444,10 @@ func TestTransform(t *testing.T) {
 			"fixtures/rule.feature",
 			"fixtures/rule.feature",
 		},
+		{
+			"fixtures/non-ascii-characters-formatting.feature",
+			"fixtures/non-ascii-characters-formatting.feature",
+		},
 	}
 
 	for _, scenario := range scenarios {

